python/nav/web/templates/foundation-5/field.html

```diff
@@ -1,46 +1,41 @@
-{% load crispy_forms_field %}
-
 {% if field.is_hidden %}
   {{ field }}
 {% else %}
   <div id="div_{{ field.auto_id }}"
-       class="ctrlHolder
-       {% if field.errors and form_show_errors %} error{% endif %}
-       {% if field|is_checkbox %} checkbox{% endif %}
-       {% if field.css_classes %} {{ field.css_classes }}{% endif %}">
+       class="ctrlHolder{% if field.errors %} error{% endif %}{% if field.css_classes %} {{ field.css_classes }}{% endif %}"
+       style="background: #BADA55 !important; padding: 4px !important;"
+  >
     {% spaceless %}
 
       {% if field.label %}
-        {% if field|is_checkbox %}
-          {% crispy_field field %}
+        {% if field.field.widget.input_type == 'checkbox' %}
```

Comment:
Instead of repeating this, we should consider implementing our own `is_checkbox` template tag.  We currently use crispy-forms 1.8.1, and as you can see, the implementation is trivial: https://github.com/django-crispy-forms/django-crispy-forms/blob/1.8.1/crispy_forms/templatetags/crispy_forms_field.py